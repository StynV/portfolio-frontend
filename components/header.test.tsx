import { render } from '@testing-library/react'
import Header from './header'

describe('Header', () => {
    it('renders', () => {
        const { container, getByText } = render(
            <Header />,
        )

        expect(container).toMatchSnapshot()

        expect(container.querySelector('header')).toBeTruthy()
        expect(container.querySelector('nav')).toBeTruthy()
        expect(getByText('Styn Vercauteren', { selector: 'span' })).toBeTruthy()
        expect(getByText('FULLSTACK DEVELOPER', { selector: 'p' })).toBeTruthy()
        expect(getByText('STYN.VERCAUTEREN@OUTLOOK.COM', { selector: 'p' })).toBeTruthy()
    })
})