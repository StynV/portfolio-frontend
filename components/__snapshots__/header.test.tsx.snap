// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders 1`] = `
<div>
  <header
    class="header"
  >
    <nav
      class="flex items-center justify-between flex-wrap pt-6 max-w-6xl"
    >
      <div
        class="headerName flex items-center flex-shrink-1"
      >
        <span
          class="font-semibold text-4xl sm:text-xl tracking-tight"
        >
          Styn Vercauteren
        </span>
      </div>
      <div
        class="headerName w-full block flex-grow lg:flex lg:items-center lg:w-auto"
      >
        <div
          class="sm:text-sm text-2xl lg:flex-grow"
        >
          <p>
            FULLSTACK DEVELOPER
          </p>
        </div>
      </div>
      <div
        class="w-full block flex-grow lg:flex lg:items-center lg:w-auto"
      >
        <div
          class="headerTextRight"
        >
          <p>
            STYN.VERCAUTEREN@OUTLOOK.COM
          </p>
        </div>
      </div>
    </nav>
  </header>
</div>
`;
